g2p/mappings/langs/moe/moe_ipa_to_eng_ipa.json

```diff
@@ -0,0 +1,594 @@
+[
```

Comment:
This moe_ipa_to_eng_ipa file does not work correctly. I'll replace it by a newly generated one. The currently recommended way to map `lang-ipa` to `eng-ipa` is to run `g2p generate-mapping --from lang --to eng` and use the automatically generated mapping. Someday, it might be worth cleaning up the manually created mappings to `eng-ipa` in crj, crl, crm and csw and replacing them by generated ones instead.